app/main/routes.py

```diff
@@ -23,6 +24,49 @@ def index():
     return redirect(url_for("categories.index"))
 
 
+@bp.get("/start")
+def start():
+    """This is the main entry point for the service from www.gov.uk/check-legal-aid
+    The Welsh version of this page directs the user to /start?locale=cy_GB, we need to set their locale cookie accordingly
+    """
+    session.clear()
+
+    response = redirect(url_for("categories.index"))
+    locale = request.args.get("locale")
+    if locale:
```

Comment:
What locale cookie is this setting? If the user already has this cookie from this service then why do we need to set again?